client/jetpack-cloud/sections/partner-portal/stored-credit-card/style.scss

```diff
@@ -0,0 +1,54 @@
+.stored-credit-card {
+	display: flex;
+	flex-wrap: wrap;
+	width: 270px;
+	height: 150px;
+	background: var( --studio-white );
+	border-radius: 2px;
+	padding: 1rem;
+}
+
+.stored-credit-card__primary {
+    padding: 2px 4px;
+	line-height: 1.25rem;
+	background: var( --studio-gray-80 );
+	color: var( --studio-white );
+	font-size: $font-body-small;
+    border-radius: var( --jetpack-corners-soft );
+    font-weight: 400;
+}
+
+.stored-credit-card__header {
+	display: flex;
+	justify-content: space-between;
+	flex-grow: 1;
+	margin-bottom: auto;
+}
+
+.stored-credit-card__payment-logo svg {
+	filter: grayscale( 100% );
+}
+
+.stored-credit-card__footer {
+	width: 100%;
+	display: flex;
+	flex-grow: 1;
+	justify-content: space-between;
+}
+
+.stored-credit-card__footer-left,
+.stored-credit-card__footer-right {
+	align-self: flex-end;
+}
+
+.stored-credit-card__name {
+	font-size: 1.125rem; /* stylelint-disable-line */
+	color: var( --studio-gray-80 );
+	margin-bottom: 1rem;
+}
+
+.stored-credit-card__number,
+.stored-credit-card__expiry {
+	font-size: $font-body-small;
+	color: var( --studio-gray-80 );
+}
```

Comment:
This is more of a question: do we follow any EOL here practise? 😅 
This is also "missing" in `client/jetpack-cloud/sections/partner-portal/add-stored-credit-card/style.scss` but exists in the rest of the files.

Reply:
There's no strict rule about it, but we usually add an empty line to the EOF